Python/sort/README.md

```diff
@@ -1 +1,26 @@
-a
+ ## Sorting Algorithms
+
+In computer science, a sorting algorithm is an algorithm that puts elements of a list in a certain order. The most frequently used orders are numerical order and lexicographical order. Efficient sorting is important for optimizing the efficiency of other algorithms that require input data to be in sorted lists. 
+
+> See: [Sorting Algorithms](https://en.wikipedia.org/wiki/Sorting_algorithm)
```

Comment:
Since it is the beginning of the markdown file, can you please keep it under the heading tag. 
Rest everything seems fine.

Reply:
The reference link?